src/game/objects/creatures/ape.c

```diff
@@ -31,7 +31,7 @@
 #define APE_RADIUS (WALL_L / 3) // = 341
 #define APE_SMARTNESS 0x7FFF
 
-typedef enum {
+typedef enum APE_ANIM {
```

Comment:
Breaks consistency with other enums, also I don't see it used anywhere. Let's revert it

Reply:
Oops sorry. For some reason in VS Code, this was needed to fix the intellisense. Without it, it looks like this:
![image](https://github.com/LostArtefacts/Tomb1Main/assets/81546780/9e48db5d-789a-4c0f-a8a7-b7d60bf7a04d)


And the types.h enums had it but I now notice other creature files don't.
```
typedef enum GAME_STATUS {
    GMS_IN_GAME = 0,
    GMS_IN_INVENTORY = 1 << 0,
    GMS_IN_PAUSE = 1 << 1,
    GMS_IN_STATS = 1 << 2,
    GMS_IN_INVENTORY_HEALTH = 1 << 3,
} GAME_STATUS;
```

Reply:
Let's revert this for now and create a follow-up task to supply this for all enums.